pyterrier/batchretrieve.py

```diff
@@ -98,8 +96,7 @@ class BatchRetrieve(BatchRetrieveBase):
     Use this class for retrieval by Terrier
     """
 
-    #from_dataset = staticmethod(partial(_from_dataset, clz=BatchRetrieve))
-    @staticmethod
+        @staticmethod
```

Comment:
indentation looks off here? (is this tabs vs spaces?)

Reply:
yes, I spotted. and ran unit tests before pushing again. this should be fixed now.